File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/BatchListResponse.java
package com.sdkwork.ai.model;

public class BatchListResponse {
    private String object;
    private List<BatchResponse> data;
    private Boolean hasMore;
    private String firstId;
    private String lastId;

    public String getObject() {
        return this.object;
    }
    
    public void setObject(String object) {
        this.object = object;
    }

    public List<BatchResponse> getData() {
        return this.data;
    }
    
    public void setData(List<BatchResponse> data) {
        this.data = data;
    }

    public Boolean getHasMore() {
        return this.hasMore;
    }
    
    public void setHasMore(Boolean hasMore) {
        this.hasMore = hasMore;
    }

    public String getFirstId() {
        return this.firstId;
    }
    
    public void setFirstId(String firstId) {
        this.firstId = firstId;
    }

    public String getLastId() {
        return this.lastId;
    }
    
    public void setLastId(String lastId) {
        this.lastId = lastId;
    }
}

File: sdkwork-ai-sdk-csharp/Models/BatchListResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class BatchListResponse
    {
        public string? Object { get; set; }
        public List<BatchResponse>? Data { get; set; }
        public bool? HasMore { get; set; }
        public string? FirstId { get; set; }
        public string? LastId { get; set; }
    }
}
